### PR TITLE
Update AbstractParseTreeVisitor.h

### PR DESCRIPTION
**Pull request by kohlsalem**
defaultResult must return nullptr not (int)0 otherwise isNotNull from class Any does not work.

[cut]
Thank you for proposing a contribution to the ANTLR project. In order to accept changes from the outside world, all contributors must "sign" the  [contributors.txt](https://github.com/antlr/antlr4/blob/master/contributors.txt) contributors certificate of origin. It's an unfortunate reality of today's fuzzy and bizarre world of open-source ownership.

Make sure you are already in the contributors.txt file or add a commit to this pull request with the appropriate change. Thanks!
[/cut]

### CODE REVIEW

File: runtime/Cpp/runtime/src/tree/AbstractParseTreeVisitor.h [hunk 1]
```diff
@@ -70,7 +70,9 @@ namespace tree {
     /// </summary>
     /// <returns> The default value returned by visitor methods. </returns>
     virtual antlrcpp::Any defaultResult() {
-      return 0;
+      //return 0;
+			//CH2017-2-2 we must return nullptr not (int)0 otherwise isNotNull from class Any does not work.
```
Comment (mike-lischke) [inline runtime/Cpp/runtime/src/tree/AbstractParseTreeVisitor.h]: Can you please fix the indentation? Otherwise the patch is ok.

Comment (kohlsalem) [inline runtime/Cpp/runtime/src/tree/AbstractParseTreeVisitor.h]: Upps - however that happened. Fixed.

**Comment by kohlsalem** [comment]
removed - this is rather a style question; but yours is counting :-)

**Comment by mike-lischke** [comment]
:-) You closed the pull request, so Ter cannot merge it anymore.

**Comment by kohlsalem** [comment]
Kopf-->Tisch 
"Ich mag Github" wäre jetzt eine leichte Übertreibung

**Comment by parrt** [comment]
@kohlsalem can you sign the contributors.txt file?

**Comment by parrt** [comment]
@kohlsalem can you sign the contributors.txt file?

**Comment by parrt** [comment]
closing unless we get a signature

**Comment by kohlsalem** [comment]
What exactly do you want me to do?

**Comment by parrt** [comment]
uh read your own PR create text above.

**Comment by kohlsalem** [comment]
sorry for not reading. done.  A lot of decoration for a one liner...

**Comment by parrt** [comment]
Yeah, it's onerous but I have to keep the license squeaky clean! :)

**Comment by kohlsalem** [comment]
well, ok for me. Kind of at least :-)  - if there is anything ele i have to do, please tell me, since this PR is still open and i thought you have to merege and Close it...

**Comment by parrt** [comment]
hmm...i don't see your commit

**Comment by kohlsalem** [comment]
did a new one

**Comment by mike-lischke** [comment]
@kohlsalem You can simply add the contributors.txt change to this PR. If you open a new one it will be disconnected from this one, which is not good. When you create  PR from a clone you just can keep adding stuff there and this will automatically be included in the PR then, until it's merged.

**Comment by kohlsalem** [comment]
Well, thats what i tried and failed upon.

**Comment by parrt** [comment]
@mike-lischke i'll just make the change (w/o comment as I don't own it).

**Pull request closed without merging**